### PR TITLE
Merge the two schedulers.

### PR DESCRIPTION
**Pull request by floitschG**
Supersedes #455.

This PR modifies as little as possible (although I couldn't resist for tiny comments and such). It mainly moves code around.

The names are not perfect, and I have a follow-up proposal.

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/animation/scheduler.dart [hunk 1]
```diff
@@ -2,157 +2,4 @@
...
-    if (_haveScheduledVisualUpdate)
-      return;
-    ui.window.scheduleFrame();
-    _haveScheduledVisualUpdate = true;
-  }
-}
-
-/// A singleton instance of Scheduler to coordinate all the callbacks.
-final Scheduler scheduler = new Scheduler._();
+export '../scheduler/scheduler.dart';
```
Comment (abarth) [inline packages/flutter/lib/src/animation/scheduler.dart]: Please remove this file in a followup patch.

File: packages/flutter/lib/src/scheduler/scheduler.dart [hunk 2]
```diff
@@ -3,16 +3,35 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:ui' show VoidCallback;
+import 'dart:developer';
```
Comment (Hixie) [inline packages/flutter/lib/src/scheduler/scheduler.dart]: Please don't leave a dart:developer import. :-)

Comment (floitschG) [inline packages/flutter/lib/src/scheduler/scheduler.dart]: It's needed for the `Timeline` calls. (I just copied it from the animation scheduler).

Comment (Hixie) [inline packages/flutter/lib/src/scheduler/scheduler.dart]: oh, wow, ok. Interesting. Nevermind then!

**Pull request merged**